### PR TITLE
with spinner + new scan form partial

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
- (#137) loader graphic appears in form button after a valid email address has been submitted/while data is loading
- (#147) consolidates email-scan form into partial, tweaks to css for different button styles in "what to do OL"

### CODE REVIEW

File: public/css/app.css [hunk 1]
```diff
@@ -1,85 +1,73 @@
-html,
-body {
+
+
+html, body {
```
Comment (pdehaan) [inline public/css/app.css]: We merged a Stylelint PR with some refactored CSS earlier today. We may need to rebase this against **master**.

File: public/js/test.js [hunk 2]
```diff
@@ -60,6 +60,15 @@ function removeUser() {
 //   }
 // }
 
+function removeLoader(){
+  if(document.getElementsByClassName("input-group-button")[0].classList.contains("loading-data")){
```
Comment (pdehaan) [inline public/js/test.js]: Do we need to add any checks that `document.getElementsByClassName("input-group-button")` is non-null before trying to access the `[0]` index?

Also, not sure if we want to try caching the "input-group-button" value so we don't need to call it 2-3 times.

File: views/layouts/default.hbs [hunk 3]
```diff
@@ -7,8 +7,8 @@
     <link href="https://fonts.googleapis.com/css?family=Fira+Sans:300,400,500,600" rel="stylesheet">
     <link rel="stylesheet" href="/css/vendor/foundation.min.css">
     <link rel="stylesheet" href="/css/app.css">
-    <script type="text/javascript" src="/js/vendor/jquery.js" defer></script>
-    <script type="text/javascript" src="/js/vendor/foundation.min.js" defer></script>
+    {{!-- <script type="text/javascript" src="/js/vendor/jquery.js" defer></script>
+    <script type="text/javascript" src="/js/vendor/foundation.min.js" defer></script> --}}
```
Comment (pdehaan) [inline views/layouts/default.hbs]: Are we not using Foundation anymore? Or are we still just using it for the CSS?

File: public/js/test.js (hunk 2 above)
Comment (lesleyjanenorton) [reply public/js/test.js]: Just pushed with the event listener wrapped in a conditional that checks for the form but I could add another conditional here if you think that would be better.

File: views/partials/scan_form.hbs [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+      <div class="input-group">
+          <input class="input-group-field email-to-hash" type="email" name="email" placeholder="Enter Email Address">
+          <div class="input-group-button">
+              <img src="img/Search.svg" alt="search icon" />
+              <img class="loader" src="img/loader.gif" alt="loading data" />
+              <input id="submit-email" type="submit" class="button" value="Search">
+          </div>
+      </div>
+      <span>Your email will not be stored. Find out more in our <a href="
+      https://www.mozilla.org/privacy/firefox/" alt="Mozilla Firefox privacy policy">Privacy Policy.</a></span>
```
Comment (pdehaan) [inline views/partials/scan_form.hbs]: Do we need an `alt=""` on the `<a>` tag?

**Comment by lesleyjanenorton** [comment]
WIP! working out logo png / svg discrepancies.

**Comment by pdehaan** [comment]
Not sure if you want to try tackling the PNG/SVG issues in a separate PR, since I can see it taking a while to resolve all logos and making sure they work (and we may not want to delay your other excellent commits due to my logo nits).

File: public/img/logos/Adobe.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns='http://www.w3.org/2000/svg' x='0px' y='0px' width='100px' height='85.811px' viewBox='0 0 100 85.811' xml:space='preserve'><path fill-rule='evenodd' fill='#ED1C24' d='M49.608,32.625c0.104,0.105,23.395,53.232,23.373,53.186c-5.098,0-10.174,0-15.264,0 c0-0.164-7.23-17.32-7.155-17.41c-5.5,0-10.963,0-16.456,0C34.254,68.444,49.838,32.446,49.608,32.625L49.608,32.625z'/><path fill-rule='evenodd' fill='#ED1C24' d='M0,0c0.119,0,35.767,0,35.775,0C35.879-0.022,0.238,85.804,0.119,85.692 C0.119,85.572,0.038-0.06,0,0L0,0z'/><path fill-rule='evenodd' fill='#ED1C24' d='M63.009,0H100c0,0.216-0.024,85.856-0.024,85.736C99.997,86.014,63.009,0,63.009,0 L63.009,0z'/></svg>
```
Comment (pdehaan) [inline public/img/logos/Adobe.svg]: I didn't see in the ocean of diffs below, but we should also confirm that we're removing the old 3 logos in public/img/ (now that we're putting logos in a nested directory):

- Adobe.png
- AllMusic.png
- Linkedin.png

File: public/img/logos/EpicGames.svg [hunk 6]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generator: Adobe Illustrator 18.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (pdehaan) [inline public/img/logos/EpicGames.svg]: Just a self-reminder that we should run all these images though svgo optimizer/minification after we merge in the logos.

File: public/img/logos/17Media.svg [hunk 7]
```diff
@@ -0,0 +1 @@
...7c-15.9,7-34.4,5.5-50.3-0.4c-10.5-3.8-20.1-11.5-24-22.2C-1.5,213.1-0.2,200,2.5,188 c4.3-19.6,13.9-38.6,29.2-51.9C38.6,130.1,46.8,125.4,55.8,123.4L55.8,123.4z"/><path fill="#FFFFFF" d="M607.7,123.3c13.1-3,26.9,1.6,37.4,9.5c14.7,11,24.8,27.3,30.4,44.7c4.3,13.8,7.2,28.7,3.8,43 c-2.4,10.7-10.3,19.5-20.1,24.3c-13.3,6.4-28.6,8.9-43.2,6.6c-13.9-2.1-26.6-10.7-33.8-22.8c-8.9-14.9-11.8-32.7-11.2-49.9 c0.6-14.2,4.6-28.6,13.3-40C590.1,131.2,598.3,125.3,607.7,123.3L607.7,123.3z"/><path fill="#FFFFFF" d="M181.2,272.6c7.4-4.8,18.6,1.4,18.6,10.2c-0.9,10.3-5.9,19.9-12.8,27.4c8.9,21.9,25.9,39.7,45.7,52.1 c23,14.4,49.4,22.8,76.2,26.6c42.2,5.9,86.5,0.1,125-18.6c25.9-12.6,49.1-33.1,60-60.2c-6.8-7.5-11.9-16.9-12.8-27.1 c-0.3-9.1,11.6-15.6,18.9-10.2c4.1,2.9,3.4,8.5,5.5,12.7c3.8,9.7,13.2,16.9,23.6,17.7c3.8,0.5,8.3-0.7,11.6,2c6.3,5,5.1,16.3-2,19.9 c-2.8,1.7-6.2,1.3-9.3,1.3c-5.5,0-10.8-1.4-15.9-3.2c-12.8,29.7-38.2,52.3-66.6,66.7c-41.1,21.1-88.6,27.9-134.2,23 c-21.5-2.4-42.7-7.6-62.7-15.8c-21.7-8.9-42.2-21.5-58.6-38.5c-9.9-10.3-18.1-22.3-23.8-35.5c-7.3,2.6-15.1,3.9-22.8,2.9 c-8.9-1.9-11.6-15.1-4.7-20.8c3.5-3,8.4-1.5,12.5-2.2c10.3-1.2,19.4-8.4,23-18.1C177.3,280.9,176.9,275.3,181.2,272.6L181.2,272.6z" /></svg>
```
Comment (pdehaan) [inline public/img/logos/17Media.svg]: Not sure how to best solve this (apart from manually redo some logos), but I believe the logos are currently being displayed against a white background, so in certain cases, these transparent+white logos will be completely invisible.

https://fx-breach-alerts.herokuapp.com/?breach=17Media (image is currently broken, but this just shows how the image will currently be rendered. You can see the "?breach=Adobe" for a non-404-ing image)

**Comment by pdehaan** [comment]
Unsurprisingly, this will need a rebasin' now that #146 has landed.

File: public/img/logos/EpicGames.svg (hunk 6 above)
Comment (pdehaan) [reply public/img/logos/EpicGames.svg]: Did you modify these image assets at all, or was this a pure copy-pasta from the **blurts-addon** repo images?

**Comment by groovecoder** [comment]
Superseded by https://github.com/mozilla/blurts-server/pull/151 with conflicts resolved.

**Pull request closed without merging**